mobilecoind/src/monitor_store.rs

```diff
@@ -230,79 +238,6 @@ impl MonitorStore {
             Err(err) => Err(Error::LMDB(err)),
         }
     }
-
-    /*
```

Comment:
This commented out stuff isn't necessary.

Reply:
Is the plan to remove it?

Reply:
Its just a bunch of commented out stuff that was leftovers from Robb's version when I was building this. No need to keep it around.